labs-ios-starter/Model/Apollo/Web Auth/Member.swift

```diff
@@ -25,15 +36,16 @@ struct Member: Equatable, Codable {
     ///   - password: optional, used for login and updating password only (defaults to nil)
     ///   - firstName: optional, used for registration and updating name only (defaults to nil)
     ///   - lastName: optional, used for registration and updating name only (defaults to nil)
-    init(identifier: UUID? = nil, email: String? = nil, firstName: String? = nil, lastName: String? = nil) {
-        self.identifier = identifier
+    init(id: String? = nil, email: String? = nil, firstName: String? = nil, lastName: String? = nil, avatarURL: URL? = nil) {
```

Comment:
should these be required parameters not optionals? id, firstname, pass

Reply:
It's ok if they're optional, not sure what's required on the backend and what's optional, so some keys could be missing